arangod/Aql/EnumerateCollectionExecutor.cpp

```diff
@@ -173,13 +173,18 @@ std::tuple<ExecutionState, EnumerateCollectionStats, size_t> EnumerateCollection
     std::tie(_state, _input) = _fetcher.fetchRow();
 
     if (_state == ExecutionState::WAITING) {
-      return {_state, stats, 0};
+      return std::make_tuple<ExecutionState, EnumerateCollectionStats, size_t>( // tupple, cannot use initializer list due to build failure
+        std::move(ExecutionState(_state)), std::move(stats), 0 // rvalue for first argument and std::move(...) required tmatch constructor signature
+      );
     }
 
     if (!_input) {
       TRI_ASSERT(_state == ExecutionState::DONE);
-      return {_state, stats, 0};
+      return std::make_tuple<ExecutionState, EnumerateCollectionStats, size_t>( // tupple, cannot use initializer list due to build failure
+        std::move(ExecutionState(_state)), std::move(stats), 0 // rvalue for first argument and std::move(...) required tmatch constructor signature
```

Comment:
```suggestion
        ExecutionState(_state), std::move(stats), 0 // rvalue for first argument and std::move(...) required tmatch constructor signature
```

Reply:
I would think so too, but MSVC was complaining with:
cannot convert argument 1 from 'arangodb::aql::ExecutionState' to 'arangodb::aql::ExecutionState &&'